content/en/blog/istio-resiliency-patterns/index.md

```diff
@@ -0,0 +1,542 @@
+
+By default Istio does not install any of the services automatically. You have to manually install Prometheus, Grafana, and Jaeger.
+
+Let's start by installing the Prometheus addon first. Open the folder where you downloaded Istio and run:
+
+```sh
+$ kubectl apply -f samples/addons/prometheus.yaml
+serviceaccount/prometheus created
+configmap/prometheus created
+Warning: rbac.authorization.k8s.io/v1beta1 ClusterRole is deprecated in v1.17+, unavailable in v1.22+; use rbac.authorization.k8s.io/v1 ClusterRole
+clusterrole.rbac.authorization.k8s.io/prometheus created
+Warning: rbac.authorization.k8s.io/v1beta1 ClusterRoleBinding is deprecated in v1.17+, unavailable in v1.22+; use rbac.authorization.k8s.io/v1 ClusterRoleBinding
+clusterrolebinding.rbac.authorization.k8s.io/prometheus created
+service/prometheus created
+deployment.apps/prometheus created
+```
+
+Wait for the Prometheus Pod to start (`kubectl get po -n istio-system` to check the status) and then you can open the Prometheus dashboard using this command: 
+
+```sh
+istioctl dashboard prometheus
+```
+
+From the UI, you can execute a Prometheus query as shown in the figure below: 
+
+
+![Prometheus Homepage](prometheus-homepage.png)
+
+You can press CTRL+C in the terminal where you executed the `dashboard prometheus` command to close the tunnel. Just like you installed Prometheus, you also need to install Grafana with a similar command:
+
+```sh
+$ kubectl apply -f samples/addons/grafana.yaml
+```
+
+To open the Grafana dashboard use the command below:
+
+```sh
+istioctl dashboard grafana
+```
+
+The browser will automatically open at the Grafana's home page that looks similar to the figure below. 
+
+
+![Grafana Homepage](grafana-homepage.png)
+
+### Istio Dashboards
+
+There are a couple of dashboards that Istio creates automatically on installation. The table below lists all the dashboards with their short descriptions.
+
+| Dashboard Name                 | Description                                                  |
+| ------------------------------ | ------------------------------------------------------------ |
+| Istio Service Dashboard        | Shows detailed graphs and telemetry for each service running inside the mesh. You can use the drop-down at the top of the dashboard to select the service from the mesh. |
+| Istio Workload Dashboard       | Shows detailed graphs and telemetry for service workloads.   |
+| Istio Mesh Dashboard           | Shows an overview of the services inside the mesh, their latency, success rate, overall request volume and number of non-5xx, 4xx, 5xx HTTP responses. |
+| Istio Performance Dashboard    | Shows the performance-related graphs for different Istio components (proxies, gateways, etc.) |
+| Istio Mixer Dashboard          | Shows resource usage graphs for Istio mixer component and adapters. |
```

Comment:
the mixer dashboard is not in Grafana anymore. There's the "Istio Wasm Extension Dashboard" that got added.